Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchController.cs

```diff
@@ -14,6 +14,23 @@ public class UnityTouchController : BaseController
         public UnityTouchController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
                 : base(trackingState, controllerHandedness, inputSource, interactions)
         {
+            if (InputSystem?.InputSystemProfile.GesturesProfile != null)
```

Comment:
Question: Does this code not work properly if left in SetupDefaultInteractions()?

Reply:
Not the original submitter, but I've been testing this and it seems like it works fine when left in SetupDefaultInteractions().